riemann_theta/riemann_theta.pyx

```diff
@@ -1000,20 +1036,19 @@ def Rbound_deriv(Y, N, tol):
         sage: Y = matrix(RR,2,2,[1,0,0,1])
         sage: Rbound_deriv(Y,3,10^(-10))
         6.6689474473...
-
     """
     cdef RealField_class RR
-    cdef RealNumber rho, Rmin, R, fR, dfR, fRo, half_g, pi
```

Comment:
Why is the declaration for half_g removed? As far as I can see, that variable is used.

Reply:
sorry, but used where ? not in this function, it seems to me. But maybe I am not fully awake.

Reply:
In these lines:

https://github.com/nbruin/RiemannTheta/blob/6a6feafbb175e71f3cf51ed8b2457583789b0fbd/riemann_theta/riemann_theta.pyx#L991
https://github.com/nbruin/RiemannTheta/blob/6a6feafbb175e71f3cf51ed8b2457583789b0fbd/riemann_theta/riemann_theta.pyx#L996

the value is initialized to be used in the local function f (as a constant there). I don't know how good cython is in special-casing expressions for declared cython types, but ideally it would take some shortcuts in the expression evaluation if it knows the type of `half_g`.

Reply:
Oh wait ... the review tools on github are really poor in displaying context. It turns out the removal happened in `Rbound_deriv` whereas the usage above is in `Rbound`. In `Rbound_deriv` it looks like this is indeed an unused declaration.

I was hoping reviewing changes would be easy on github, but clearly it's still vastly inferior to a `git diff` with a good quality multi-way diff/merge tool like `meld` or similar.

This merge request should be OK and can be merged.